client/src/partials/templates/bhDateEditor.tmpl.html

```diff
@@ -31,11 +31,10 @@
         is-open="$ctrl.editMode"
         datepicker-options="$ctrl.options"
         show-button-bar="false"
-        datepicker-append-to-body="true"
         data-date-editor-input
         ng-readonly="!$ctrl.editMode"
         ng-disabled="$ctrl.disabled"
-        required>
```

Comment:
This change will break some pages!  Could you use `ng-required` (see [documentation](https://docs.angularjs.org/api/ng/directive/ngRequired)) instead, and have it required by default?  That way only controllers do not need required can turn it off.